examples/bigquery/billing_account/main.tf

```diff
@@ -28,8 +28,9 @@ module "log_export" {
 }
 
 module "destination" {
-  source                   = "../../..//modules/bigquery"
-  project_id               = "${var.project_id}"
-  dataset_name             = "bigquery_example"
-  log_sink_writer_identity = "${module.log_export.writer_identity}"
+  source                     = "../../..//modules/bigquery"
+  project_id                 = "${var.project_id}"
+  dataset_name               = "bigquery_example"
+  log_sink_writer_identity   = "${module.log_export.writer_identity}"
+  delete_contents_on_destroy = "true"
```

Comment:
We can leave it at a `true` default.

Reply:
Is it really good idea to delete the contents as a default behavior?

Reply:
I think we could discuss changing the default later, but it'd be good to release this without making it backwards-incompatible.

Reply:
Sure makes sense, let me change it for now